lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +20,39 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    file = open(text)
+    en_text = file.read()
+    en_text_list = []
+    en_text_list = en_text.split()
+    words_list = []
+
+    for i in range(len(en_text_list)):
```

Comment:
try to find all the letter symbols in the text, this will help remove punctuation marks immediately